static/src/stylesheets/layout/new-header/_feedback-form.scss

```diff
@@ -19,6 +19,12 @@
 
     @include mq(wide) {
         width: $gs-column-width * 3;
+
+        .has-page-skin & {
+            width: auto;
+            position: relative;
+            left: 0;
```

Comment:
![tenor](https://user-images.githubusercontent.com/2244375/27800491-83764a20-6011-11e7-9f42-4859edceb162.gif)

Ah oh! Maybe add a `right: auto` for clearification?